Automattic/wp-calypso
# Stats: Fixing focus for new date control

Related to #81936 

## Proposed Changes

* update text input location to avoid losing focus when typing

## Testing Instructions



* open the live branch and go to the Stats
* apply feature flag `?flags=stats/date-control`
* verify that the input fields in the new control don't lose focus when you type new dates

## Pre-merge Checklist
